pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java

```diff
@@ -192,18 +191,15 @@ private void closePingFuture() {
     @Override
     public void onWebSocketConnect(Session session) {
         super.onWebSocketConnect(session);
-        WebSocketProxyConfiguration webSocketProxyConfig = service.getWebSocketProxyConfig();
-        if (webSocketProxyConfig != null) {
-            int webSocketPingDurationSeconds = webSocketProxyConfig.getWebSocketPingDurationSeconds();
```

Comment:
Does the field `webSocketPingDurationSeconds` in class `WebSocketProxyConfiguration` not useful now? If yes, should we delete it?

Reply:
No, these configurations can be converted to each other, so add `webSocketPingDurationSeconds` to each config.

Reply:
I see.

> Should we also should edit the test added in https://github.com/apache/pulsar/pull/19203?

And this comment ^_^

Reply:
No. My changes don't break the tests of #19203.